doc/nyan/engine_types.md

```diff
@@ -50,6 +49,10 @@ Unit(): # WIP
 	# Everything that has a location ingame.
 
 	hp : float
+	idleLook : animation
+	death : Unit # What is the dead form of this Unit?
+	# Proposal:
+#	dying : animation
```

Comment:
as noted in the Deer-discussion above, not every unit has the ability to die. then we'd have to insert a placeholder or an empty value for `dying`, which we wanna avoid.